src/js/themesBrandEvolution/darkTheme/colors.ts

```diff
@@ -0,0 +1,77 @@
+import { brandColor } from '../../brandColorBrandEvolution';
+import type { ThemeColors } from '../types';
+
+export const colors: ThemeColors = {
+  background: {
+    default: brandColor.grey800, // For default neutral backgrounds
```

Comment:
We many not need these comments if they are present in the types. I think it's the types that enables them to display on hover in Visual Studio